pkg/config/config.go

```diff
@@ -13,30 +13,32 @@ import (
 	"k8s.io/klog/v2"
 )
 
-const COMPONENT_VERSION = "2.5.0"
+const COMPONENT_VERSION = "2.7.0"
 
 var DEVELOPMENT_MODE = false // Do not change this. See config_development.go to enable.
 var Cfg = new()
 
 // Struct to hold our configuratioin
 type Config struct {
+	DBBatchSize    int // Batch size used to write to DB.
 	DBHost         string
+	DBMaxConns     int // Max size of DB connection pool.
```

Comment:
Renamed to keep all DB settings consistent and avoid confusion with the similar `RequestLimit` added with this PR.